### ISSUE TITLE
[16.0] module: description

### ISSUE THREAD

**Issue by wm-cortex** [odoo/odoo#184213]
### Odoo Version

- [X] 16.0
- [ ] 17.0
- [ ] 18.0
- [ ] Other (specify)

### Steps to Reproduce

**Current behaviour:**
Install l10n_sa_edi module and configure the connection to ZATCA
Create a credit note with line discount 
Confirm and process the credit note. The credit note sent to ZATCA successfully with a below warning:

 Invoice line net amount (BT-131) must equal (Invoiced quantity (BT-129) * (Item net price (BT-146) / item price base quantity (BT-149))) + Sum of invoice line charge amount (BT-141) - Sum of invoice line allowance amount (BT-136)

**Comment by Atteqany** [comment]
Did you find the solution bro ?

**Comment by wm-cortex** [comment]
@Atteqany No, I just follow this workaround https://github.com/odoo/odoo/issues/168587#issuecomment-2425552173
